beacon-chain/rpc/prysm/v1alpha1/validator/proposer_bellatrix.go

```diff
@@ -35,19 +37,27 @@ var builderGetPayloadMissCount = promauto.NewCounter(prometheus.CounterOpts{
 
 var gweiPerEth = big.NewInt(int64(params.BeaconConfig().GweiPerEth))
 
+// emptyTransactionsRoot represents the returned value of ssz.TransactionsRoot([][]byte{}) and
+// can be used as a constant to avoid recomputing this value in every call.
+var emptyTransactionsRoot = [32]byte{127, 254, 36, 30, 166, 1, 135, 253, 176, 24, 123, 250, 34, 222, 53, 209, 249, 190, 215, 171, 6, 29, 148, 1, 253, 71, 227, 74, 84, 251, 237, 225}
+
```

Comment:
While i was here, i moved this to a global so we don't have to recompute it every time. I've also added a test to ensure this value is correct